blazor/autocomplete/filtering.md

```diff
@@ -210,4 +210,73 @@ The AutoComplete component filter queries can be customized. You can also use yo
         await autoObj.FilterAsync(Country, query);
     }
 }
-```
\ No newline at end of file
+```
+
+## Properties
+
+### FilterType
+
+Determines on which filter type, the component needs to be considered on search action.
```

Comment:
Determines which filter type, the component needs to be considered during search action.